.gitignore

```diff
@@ -2,3 +2,5 @@ node_modules/
 .DS_Store
 .cache/
 public/
+.parcel-cache
```

Comment:
also updated to gitignore these files on start locally. I used local because codesandbox was having some performance issues for me, possibly due to brave